core/subsurface-qt/SettingsObjectWrapper.cpp

```diff
@@ -7,6 +7,7 @@
 #include <QNetworkProxy>
 
 #include "core/qthelper.h"
+#include "../prefs-macros.h"
```

Comment:
I know we are not consistent in this, but as you work towards consistency, would you mind adding header file references relative to $SRC - so `core/prefs-macros.h` instead of `../prefs-macros.h` ?

Reply:
Good point, changed.